src/js/contentscripts/dnt.js

```diff
@@ -20,28 +20,34 @@ function getPageScript() {
   // code below is not a content script: no chrome.* APIs /////////////////////
 
   // return a string
-  return "(" + function (NAVIGATOR, OBJECT) {
```

Comment:
These need to be here, or `NAVIGATOR` and `OBJECT` are undefined.

Reply:
oops, thanks for catching this